controller/api/destination/server.go

```diff
@@ -100,6 +102,12 @@ func (s *server) Get(dest *pb.GetDestination, stream pb.Destination_GetServer) e
 		log,
 	)
 
+	var token contextToken
+	b := dest.GetContextToken()
+	if err := json.Unmarshal([]byte(b), &token); err != nil {
```

Comment:
I'm thinking about the upgrade story and I think we need to be able to accept tokens in either format: `ns:<namespace>` or json.  During an upgrade, the destination controller will get upgraded first but the proxies will still be running the previous version and will still be sending context tokens in the old format.

Reply:
Amazing suggestion, I was a bit short sighted and did not think about backwards compatibility that much. Thank you!